baseledger/ops/deploy_testnet_node.md

```diff
@@ -0,0 +1,58 @@
+https://docs.cosmos.network/master/run-node/run-node.html
+
+
+# Preparation
+
+DEVS Deploy contracts to testnet and make sure correct values configured in binaries
+DEVS run starport chain build locally for baseledger node
+DEVS run cargo build --release locally for orchestrator 
+DEVS Prepare the package of the compiled binaries to be shared with other node operators (BASELEDGER_PACKAGE)
+
+# Initialization
+
+Finspot node:
+
+1. Install latest golang
+2. Install cosmovisor: Follow the *deployment_and_chain_upgrade.md*
+3. Place binaries in the respective folders as described in *deployment_and_chain_upgrade.md*
+4. Initialize: cosmovisor init validator --chain-id=baseledger
+5. Configure genesis: Edit the genesis file for various params (voting time, tokens metadata etc.)
+6. Generate validator account: cosmovisor keys add --keyring-backend test validator (make sure to write down the address and the mnemonic)
+7. Generate orchestrator account: cosmovisor keys add --keyring-backend test orchestrator (make sure to write down the address and the mnemonic)
+8. Add faucet account with allocation: cosmovisor add-genesis-account --keyring-backend test baseledger1xgs5tamqre7rkz5q7d5fegjsdwufxxvt36w0a0 10000000000work
+9. Add validator account with allocation: cosmovisor add-genesis-account --keyring-backend test <validator_address> 10000000000stake,10000000000work
```

Comment:
should we give validator so much power? maybe 1 stake, 0 work?

Reply:
let's fix today during testing